fedora_elections/__init__.py

```diff
@@ -180,6 +180,13 @@ def rjust_filter(text, length):
     """
     return str(text).rjust(length)
 
+@APP.template_filter('avatar')
+def avatar_filter(openid, size=64, default='retro'):
+    import urllib
+    import hashlib
```

Comment:
For PEP-8, these should be moved to the top of the module with the other import statements.